debian/rules

```diff
@@ -38,7 +38,12 @@ override_dh_installdocs:
 	dh_installdocs -O--buildsystem=python_distutils
 
 override_dh_installinit:
-	dh_installinit -O--buildsystem=python_distutils --name rtslib-fb-targetctl
+	dh_installinit \
+		--no-start \
+		--no-enable \
```

Comment:
What would start/enable the service if we set --no-enable and --no-start?

It would be useful to see what the resulting .postrm and .postinst files look like.

Reply:
> What would start/enable the service if we set --no-enable and --no-start?

IMO, we should be explicit about this, and enable the service in "delphix-platform" or the virtualization application. If we don't do this now, I can open a PR to start doing this.

Reply:
Yeah, I agree that it's better to have the enable moved to delphix-platform (we don't do it right now). We probably don't need a start method as we expect the service to start automatically at boot time and not stop during an upgrade.